src/style.css

```diff
@@ -1 +1,24 @@
-/* add styles here */
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars {
+  display: flex;
+}
+
+.container {
+  margin: 0;
+}
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). The zero margin set on the `.container` class should be specified for either top or bottom to prevent potential margin collapse issues.